### ISSUE TITLE
New version

### ISSUE THREAD

**Issue by noozo** [Sebi55/hcaptcha#4]
Hi there, any plans to release new versions in hex? Last version seen there is 0.0.1. Thanks :)

**Comment by vincentfretin** [comment]
Hi, latest commit changed the version to 0.0.2 bu there is no release indeed.
In the meantime I'm using
```
{:hcaptcha, github: "sebastiangrebe/hcaptcha", tag: "77d7ea70775e3f810cc06bd3dd16c265378cccff"}
```
instead of 
```
{:hcaptcha, "~> 0.0.2"}
```

**Comment by sebastiangrebe** [comment]
Hey guys, sorry totally missed and maybe quite late for you
I have published 0.0.2 just now, let me know if you notice any issues
